spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -587,10 +391,16 @@ object Graph extends ContainerSerializable {
    * @param output output node
    * @return a graph container
    */
-  def apply[T: ClassTag](input : Array[ModuleNode[T]], output : Array[ModuleNode[T]],
-      variables: Option[(Array[Tensor[T]], Array[Tensor[T]])] = None,
+  def apply[T: ClassTag](
+    input : Array[ModuleNode[T]],
+    output : Array[ModuleNode[T]],
+    variables: Option[(Array[Tensor[T]], Array[Tensor[T]])] = None,
     generateBackward: Boolean = true)(implicit ev: TensorNumeric[T]) : Graph[T] = {
-    new Graph[T](input, output, variables, generateBackward)
+    if (hasControlOps(output)) {
```

Comment:
Instead of checking if there are ControlOps here, I think we can simply make all factory methods to return StaticGraph - that's the default behavior of module implementations in BigDL.

We can check ControlOps (or explicitly generate DynamicGrpah when needed) in TF loader.